MoltenVK/MoltenVK/GPUObjects/MVKPixelFormats.mm

```diff
@@ -1542,7 +1542,6 @@
 	// Including this here so we remember to update this if support is added to Vulkan in the future.
 	bool atomic64 = noVulkanSupport && (gpuCaps.supportsApple9 || (gpuCaps.supportsApple8 && gpuCaps.supportsMac2));
 	enableMTLPixFmtCapsIf( atomic64, RG32Uint, Atomic );
-	enableMTLPixFmtCapsIf( atomic64, RG32Sint, Atomic );
```

Comment:
Why did you delete one line, but not the other?

Reply:
Mentioned this in the description, this part being left here seems benign since it's disabled by a bool and has a comment that its left as a reminder, but according to the Metal spec 64-bit texture atomics are only supported for RG32Uint, not RG32Sint, so that had to be removed regardless.

Reply:
I can remove the whole section if you want though, since it's not being used.

Reply:
Ah, OK. Silly me. You can leave it in.